python_modules/dagster/dagster/core/workspace/workspace.py

```diff
@@ -27,7 +27,7 @@ class IWorkspace(ABC):
     """
 
     @abstractmethod
-    def get_location(self, location_name: str):
+    def get_repository_location(self, location_name: str):
```

Comment:
might as well add the type
```suggestion
    def get_repository_location(self, location_name: str) -> RepositoryLocation:
```